code/modules/mob/living/carbon/human/inspection_panel.dm

```diff
@@ -0,0 +1,39 @@
+/datum/inspection_panel
+	/// Mob that the examine panel belongs to.
+	var/mob/living/holder
+
+/datum/inspection_panel/ui_state(mob/user)
+	return GLOB.always_state
+
+/datum/inspection_panel/ui_interact(mob/user, datum/tgui/ui)
+	ui = SStgui.try_update_ui(user, src, ui)
```

Comment:
I feel like the two mobs should be able to see each other before this updates.

Reply:
I decided to let the game keep updating the panel, due to the fact it's not showing a literal live preview of their being, unlike SR.